gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -694,4 +701,22 @@ private static Project findDerivedProject(Project project, String projectName) {
             return project.getRootProject().findProject(projectName);
         }
     }
+
+    private static void buildDependsOn(Project project, TaskProvider<?> task) {
```

Comment:
I feel like I've this around a bit, should we move this to `gradle-utils`?

Reply:
Yeah, I had that thought - it could even live on EnvironmentVariables (the circle node == 0 or local bit)

Reply:
https://github.com/palantir/gradle-utils/pull/117

Reply:
Updated to use that version